cmsl1t/utils/draw.py

```diff
@@ -0,0 +1,115 @@
+from rootpy.plotting.utils import draw as r_draw
+from rootpy.plotting.hist import Efficiency
+from rootpy.plotting import Style, Canvas
+from rootpy.context import preserve_current_style
+from rootpy.ROOT import gStyle, TLatex
+import rootpy.ROOT as ROOT
+from exceptions import RuntimeError
+
+
+__known_root_pallettes = set(["DeepSea", "GreyScale",
```

Comment:
Haha, no.  I would rather not have had to do it this way, but I could see no alternative. The TColor header defines an enum for the pallettes: https://root.cern.ch/doc/master/TColor_8h.html#acb46d776ab87271c3fc10b14c50b169c
In rootpy (or pyROOT) these are accessible as strings, however they're in the global ROOT namespace (since the enum is not defined within the TColor class, just in the header file), so it would not really be possible to check that a given palette string passed to these methods is actually a root palette, since it could just be any method / class in the ROOT namespace. So I've copied and reformatted the specification of the enum from the above link as a way to validate that the requested palette actually exists in ROOT.

I'll put something along these lines into a comment in the code.